casepro/statistics/migrations/0008_dailyminutetotalcount.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('statistics', '0007_populate_label_totals'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DailyMinuteTotalCount',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('item_type', models.CharField(help_text='The thing being counted', max_length=1)),
+                ('scope', models.CharField(help_text='The scope in which it is being counted', max_length=32)),
+                ('count', models.IntegerField()),
+                ('minutes', models.IntegerField()),
```

Comment:
I agree that cases are unlikely to be opened and closed within a time frame that requires more accuracy than minutes - but if we want to re-use this class for timing other things we might need more accuracy. If you were timing bulk replies for example, you wouldn't want 59 seconds becoming 0 minutes etc.

As you say I don't think 60x factor changes much here - if we think we're anywhere near overflowing we should switch to something like `BigIntegerField`. Anyways, your call.